java/source/create.rst

```diff
@@ -185,6 +185,43 @@ Array of List
 
     [[1,2,3], [10,20,30], [100,200,300], [1000,2000,3000]]
 
+
+Splicing
+========
+
+Splicing provides a way of copying a range of rows between two vectors of the same type.
+
+Splicing IntVector
+------------------
+
+In this example, we copy a portion of the input IntVector to a new IntVector.
+
+.. testcode::
+
+    import org.apache.arrow.memory.BufferAllocator;
```

Comment:
Just one thing: everything here is indented by one space too many I think (it should line up). Same with the output below

Reply:
There is similar comment on other PR, shall fix this once the other is fixed as expected :)

Reply:
The other PR is now merged

Reply:
Done, hopefully it shall be correct this time.